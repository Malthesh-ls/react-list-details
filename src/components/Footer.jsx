const Footer = ({ title }) => {
  const currentDate = new Date();
  return (
    <div className="footer">
      &copy;
      {currentDate.getFullYear()}
      {' '}
      - {title}
    </div>
  );
}

export default Footer
