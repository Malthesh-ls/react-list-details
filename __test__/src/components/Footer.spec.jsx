import renderer from 'react-test-renderer';
import Footer from '../../../src/components/Footer';

describe('Footer Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<Footer title='test app' />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});