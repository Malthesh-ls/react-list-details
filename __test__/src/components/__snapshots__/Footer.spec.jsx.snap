// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Tests should match snapshot 1`] = `
<div
  className="footer"
>
  Â©
  2024
   
  - 
  test app
</div>
`;
